### PR TITLE
Implement - Allure 2 reporting

### PR DESCRIPTION
**Pull request by devGnode**
Implementation of allure 2 reporting :
 
https://docs.qameta.io/allure/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@ log/
 drivers/
 src/main/resources/config/*.*
 src/main/resources/config/*.template
```
Comment (l-bachmann) [inline .gitignore]: Couldyou delete the .template file from the git ignore please ; if the template changes, we need to push the new version.

File: src/main/java/fr/zenity/desbugs/Enum/UrlEnvironment.java [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
     DEVELOP,
     NONE,
+    API,
```
Comment (l-bachmann) [inline src/main/java/fr/zenity/desbugs/Enum/UrlEnvironment.java]: I created a second file for the API config but we can leave it here for the moment.

File: pom.xml [hunk 3]
```diff
@@ -116,11 +123,6 @@
       <version>${cucumberInfoCukes.version}</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
```
Comment (l-bachmann) [inline pom.xml]: Are you sure to delete this one ?

Comment (devGnode) [reply pom.xml]: Doublon

**Pull request merged**